pkg/resources/grant_privileges_to_account_role.go

```diff
@@ -815,6 +815,11 @@ func prepareShowGrantsRequestForAccountRole(id GrantPrivilegesToAccountRoleId) (
 	case OnAccountObjectAccountRoleGrantKind:
 		data := id.Data.(*OnAccountObjectGrantData)
 		grantedOn = data.ObjectType
+
+		// For EXTERNAL VOLUME, we have to change it to VOLUME because that's what Snowflake returns
```

Comment:
We have recently fixed similar issue on the SDK level: #2522. Shouldn't we also fix it here on the SDK level (by modifying `func (row grantRow) convert() *Grant {` function)?

Reply:
hmmm, true. Done

Reply:
I don't understand why :(

Reply:
I think I was thinking about mapping in a wrong way that's why I thought it couldn't be done, but with mapping VOLUME to EXTERNAL VOLUME can be done in SDK. Moved to the SDK.